monkey/common/tags/attack.py

```diff
@@ -5,6 +5,7 @@
 T1098_ATTACK_TECHNIQUE_TAG = "attack-t1098"
 T1105_ATTACK_TECHNIQUE_TAG = "attack-t1105"
 T1110_ATTACK_TECHNIQUE_TAG = "attack-t1110"
+T1135_ATTACK_TECHNIQUE_TAG = "attack-t1135"
```

Comment:
Not for this PR, but perhaps we should give these human-readable names, like `NETWORK_SHARE_DISCOVERY_T1135_TAG`